client/js/components/support/DpSupport.vue

```diff
@@ -81,20 +81,41 @@ export default {
   methods: {
     ...mapActions('customerContact', {
       fetchContacts: 'list'
-    })
+    }),
+
+    fetchCustomerContactsData () {
+      let params = {
+        fields: {
+          CustomerContact: [
+            'title',
+            'phoneNumber',
+            'text',
+            'eMailAddress',
+            'visible'
```

Comment:
visible should only be fetched if the permission is set.

Reply:
Sorry, you are right. When filtered contacts are received from the backend, this property is no longer needed here https://github.com/demos-europe/demosplan-core/pull/2300/commits/4fec05388b0f54bca521f3ad8dbeecdaf5aaf03d